jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter-interface.l

```diff
@@ -1424,7 +1427,7 @@
     (let (avs)
       (setq avs
             (list
-              (send *baxter* :approach-from-downside arm)
+              (send *baxter* :midpose-hold-opposite-hand-object-downside arm)
```

Comment:
too long name :(
i will use this midpose when it approach from down side, so `midpose-approach-from-downside` would be better
what do you think?